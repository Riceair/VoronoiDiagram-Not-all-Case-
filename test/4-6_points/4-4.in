#四點測試 垂直兼水平共線
4
100 100
100 200
50 200
150 200