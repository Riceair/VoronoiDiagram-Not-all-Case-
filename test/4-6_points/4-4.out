P 50 200
P 100 100
P 100 200
P 150 200
E -62925 -31350 75 150
E 75 150 75 31700
E 75 150 125 150
E 125 150 125 31700
E 125 150 63125 -31350
